forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -54,11 +50,26 @@ internal class ConfigHelper: ConfigInterface {
                 .putString(clientId, frOptions.oauth.oauthClientId)
                 .putString(revokeEndpoint, frOptions.urlPath.revokeEndpoint)
                 .putString(endSessionEndpoint, frOptions.urlPath.endSessionEndpoint)
+                .putString(logoutEndpoint, frOptions.urlPath.logoutEndpoint)
                 .putString(scope, frOptions.oauth.oauthScope)
                 .putString(redirectUri, frOptions.oauth.oauthRedirectUri)
                 .apply()
         }
 
+        /**
+         * Remove the persisted tokens
+         *
+         * @param context  The Application Context
+         * @param options  The FrOptions to be passed
+         */
+        @JvmOverloads
+        @JvmStatic
+         fun closeSession(context: Context,
```

Comment:
I think the ConfigHelper should focus on handling the Configuration but not closing the session. Do we need this method?